collect_app/src/main/java/org/odk/collect/android/views/AudioButton.java

```diff
@@ -132,6 +133,8 @@ public AudioHandler(FormIndex index, String selectionDesignator, String uri, int
                             .subscribe(event -> {
                                 if (anotherPlayerStarted(event) && mediaPlayer.isPlaying()) {
                                     mediaPlayer.stop();
+                                } else if (event.getEventType() == STOP_PLAYING && mediaPlayer.isPlaying()) {
+                                    stopPlaying();
```

Comment:
I believe these two branches should both use `stopPlaying` and that it would be clearer to separate then by `||`.